geomesa/src/main/scala/geotrellis/spark/io/geomesa/GeoMesaAttributeStore.scala

```diff
@@ -0,0 +1,30 @@
+package geotrellis.spark.io.geomesa
+
+import geotrellis.spark.LayerId
+
+import org.geotools.data.DataStoreFinder
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStore
+
+import scala.collection.JavaConverters._
+
+case class GeoMesaAttributeStore(instanceName: String, zookeepers: String, user: String, password: String, useMock: Boolean = false) extends Serializable {
```

Comment:
This is not an `AttributeStore` and doesn't really do anything like AttributeStore interface. lets just make it an `apply`. So maybe there is a need of an object that is like `GeoMesaConnector` that keeps the map as fields?